### PR TITLE
chore: update memberId schema to allow for strings

### PR DESCRIPTION
**Pull request by omer-tariq**
Allows for memberIds to be strings

### CODE REVIEW

File: test/schemas/store-order-recipient.test.js [hunk 1]
```diff
@@ -25,13 +25,13 @@ describe('storeOrderRecipientSchema.validate()', () => {
     expect(storeOrderRecipientSchema.validateSync(params)).toEqual(params);
   });
 
-  test('it reject an id that is an empty string', async () => {
+  test('it does not reject an id that is an empty string', async () => {
```
Comment (shamrt) [inline test/schemas/store-order-recipient.test.js:28]: This should still happen. We should not accept empty strings.

File: test/schemas/store-order-recipient.test.js [hunk 2]
```diff
@@ -25,13 +25,13 @@ describe('storeOrderRecipientSchema.validate()', () => {
...
     const params = { id: '', accept_language: testLocale };
 
-    await expect(storeOrderRecipientSchema.validate(params)).rejects.toThrowError('ID must be a number or UUID');
+    await expect(storeOrderRecipientSchema.validateSync(params)).toEqual(params);
   });
 
-  test('it reject an id that is undefined', async () => {
+  test('it should not reject an id that is undefined', async () => {
```
Comment (shamrt) [inline test/schemas/store-order-recipient.test.js:34]: Same here.

File: test/schemas/store-order-recipient.test.js [hunk 3]
```diff
@@ -61,13 +61,13 @@ describe('isNumberOrUuid', () => {
     ${[1, 2, 3]}                | ${false}
     ${null}                     | ${false}
     ${undefined}                | ${false}
-    ${-100}                     | ${false}
-    ${''}                       | ${false}
-    ${'fakeUuid'}               | ${false}
+    ${-100}                     | ${true}
```
Comment (shamrt) [inline test/schemas/store-order-recipient.test.js:64]: Not sure we'd want to accept a negative 🤔

File: test/schemas/store-order-recipient.test.js [hunk 4]
```diff
@@ -61,13 +61,13 @@ describe('isNumberOrUuid', () => {
...
+    ${''}                       | ${true}
```
Comment (shamrt) [inline test/schemas/store-order-recipient.test.js:65]:
```suggestion
    ${''}                       | ${false}
```

File: test/schemas/store-order-recipient.test.js [hunk 5]
```diff
@@ -61,13 +61,13 @@ describe('isNumberOrUuid', () => {
...
+    ${'fakeUuid'}               | ${true}
```
Comment (shamrt) [inline test/schemas/store-order-recipient.test.js:66]: We should either add another condition here, or change `'fakeUuid'` to `faker.random.alphaNumeric()` (or whatever)

**Review by shamrt — CHANGES_REQUESTED** [review]

File: package.json [hunk 6]
```diff
@@ -39,7 +39,7 @@
   "scripts": {
     "build:docs": "jsdoc -c jsdoc.json lib/rewardops.js",
     "lint": "eslint --cache --ignore-path .gitignore .",
-    "test": "jest",
+    "test": "jest --watch",
```
Comment (shamrt) [inline package.json:42]: Can you remove this, please? It's used in automations and won't terminate with the `--watch` flag

**Review by shamrt — CHANGES_REQUESTED** [review]
Looks good. Small revert, then we're good to go

File: package.json (hunk 6 above)
Comment (omer-tariq) [reply package.json:42]: Ah crap, Dont know how that got in.

**Review by shamrt — APPROVED** [review]

**Pull request merged**